src/main/java/view/ResultView.java

```diff
@@ -0,0 +1,39 @@
+package view;
+
+import java.util.List;
+
+import domain.Car;
+import domain.Cars;
+import service.CarRace;
+
+public class ResultView {
+
+    private static final String DASH = "-";
+    private static final String DELIMITER = " : ";
+
+    public void printCarLocation(Cars cars, CarRace carRace, int attempts) {
+        for (int i = 0; i < attempts; i++) {
+            convertCarLocation(carRace, cars.getCarList());
+            System.out.println();
+        }
+        System.out.println(winnerMessage(cars));
+    }
+
+    private void convertCarLocation(CarRace carRace, List<Car> carList) {
+        List<Car> list = carRace.updateCurrentLocation(carList);
+        for (Car car : list) {
```

Comment:
```suggestion
        for (Car car : cars) {
```
cars나, 의미를 더 넣고싶다면 runningCars 등 조금 더 의미있는 변수명을 쓰면 더 좋을 것 같습니다 :)